lib/install.ps1

```diff
@@ -929,13 +935,21 @@ function env_rm_path($manifest, $dir, $global, $arch) {
     $env_add_path = arch_specific 'env_add_path' $manifest $arch
     $dir = $dir.TrimEnd('\')
     if ($env_add_path) {
+        if (get_config USE_ISOLATED_PATH) {
+            $target_path = 'SCOOP_PATH'
+        } else {
+            $target_path = 'PATH'
+        }
```

Comment:
I'm think of a seamless transition of this feature. With current implementation, there are two situations I concern:

- If one turns on `use_isolated_path` and then updates one installed app, in this case the path added to `$env:PATH` by the installed version can't be removed and remains.
- If one turned on `use_isolated_path` and installed one app, then they turn off `use_isolated_path` and try to uninstall the app, in this case the path added to `$env:SCOOP_PATH` can't be removed and remains.

Both situations can be mitigated by checking `$env:PATH` and `$env:SCOOP_PATH` side by side whatever the `use_isolated_path` would be. WDYT?

Reply:
> For Path switching, I'll make a PR by using https://github.com/ScoopInstaller/Scoop/pull/5851's Complete-ConfigChange().

This is one possible solution, when the user changes the config, all paths will be moved. And of course, removing from all target env sounds reasonable.

Reply:
#5851 is not going to be included in v0.4.0 while this PR will be, so there should be a preliminary solution.

Reply:
> #5851 is not going to be included in v0.4.0 while this PR will be, so there should be a preliminary solution.

btw, #5851 is ready for review. I've been using it for several days now and everything is working well.

Reply:
Done. And I'll check if there's a more efficient way.

Reply:
Aha, okay, I could check all paths in PATH that start with $scoopdir. Will fix it later.

Reply:
Oh you probably misunderstood my point. I didn't mean to force converting all previously added PATH. I meant it should check both `$env:PATH` and `$env:SCOOP_PATH` when doing uninstallation so there won't be paths leftover when users toggle the config back and forth.

Reply:
All done. Now switching won't waste any more time and uninstalling will remove paths from `$env:PATH` and `$env:SCOOP_PATH`.